book/text.md

```diff
@@ -88,23 +89,18 @@ Yet Tk's *font objects* correspond to the older meaning of font: a
 type at a fixed size, style, and weight. For example:[^after-tk]
 
 [^after-tk]: You can only create `Font` objects, or any other kinds of
-    Tk objects, after calling `tkinter.Tk()`, which is why I'm putting
-    this code in the `Browser` constructor.
-
-``` {.python}
-import tkinter.font
-```
```

Comment:
This example/real code split will look terrible in print due to the background for examples. Also if it's example code it can't go in the `Browser` constructors. My proposal here is a totally different example code block, and then `tkinter.font` appears later.